### ISSUE TITLE
do-partition fails on 25k.fq.gz

### ISSUE THREAD

**Issue by ctb** [dib-lab/khmer#1257]
`do-partition.py -k 21 xxx data/25k.fq.gz -M 1e8` yields this error:

```
done starting threadsstarting: 
xxx 0
saving: xxx 0
Exception in thread Thread-1:
Traceback (most recent call last):
  File "/Library/Frameworks/EPD64.framework/Versions/7.3/lib/python2.7/threading.py", line 551, in __bootstrap_inner
    self.run()
  File "/Library/Frameworks/EPD64.framework/Versions/7.3/lib/python2.7/threading.py", line 504, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/Users/t/dev/ipy7/lib/python2.7/site-packages/khmer-2.0_rc2_144.g0f27619-py2.7-macosx-10.5-x86_64.egg/EGG-INFO/scripts/do-partition.py", line 58, in worker
    except queue.Empty:
AttributeError: Queue instance has no attribute 'Empty'
```

**Comment by mr-c** [comment]
I've fixed this in #1255

**Comment by mr-c** [comment]
Fixed in #1268
